src/DurableTask.SqlServer/SqlUtils.cs

```diff
@@ -26,7 +26,6 @@ static class SqlUtils
         {
             return reader.IsDBNull(columnIndex) ? null : reader.GetString(columnIndex);
         }
-
```

Comment:
nit: can we re-add this newline? I prefer to have whitespace between method declarations.